test/create_url_fetcher_test_certs.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd ${DIR}
+. generate_certs.sh
```

Comment:
Because the cert/ca scripts in that directory assume that they're run *from* that directory, and I didn't want to roll fixing that up into this PR.
I've added a `TODO` and sprinkled a few more `${DIR}` around in anticipation.

How are all the diodes down your left side doing today, BTW?